dex/msgjson/types.go

```diff
@@ -801,12 +805,39 @@ func (c *Connect) Serialize() []byte {
 	return append(s, uint64Bytes(c.Time)...)
 }
 
-// ConnectResult is the result result for the ConnectRoute request.
+// ConnectResult is the result for the ConnectRoute request.
 type ConnectResult struct {
 	Sig     Bytes    `json:"sig"`
 	Matches []*Match `json:"matches"`
 }
```

Comment:
Maybe a TODO to include the penalty object, if user is penalized?

Reply:
Yes, a TODO here for that would be great.  And maybe we could keep the auth.Notify temporarily if the TODO mentions to change it to a regular send.